Add STAKEWORLD

stakeworld opened the pull request.

rossbulat commented:
Hi Stakeworld, thank you for submitting your details. Could you provide a native SVG image as your thumbnail? Right now it is a PNG wrapped in an SVG.

stakeworld commented:
> Hi Stakeworld, thank you for submitting your details. Could you provide a native SVG image as your thumbnail? Right now it is a PNG wrapped in an SVG.

Hi, I think i've updated the pull request with a native SVG, please let me know if it is ok now. I've added it to this reply just to be sure.
![stakeworld](https://user-images.githubusercontent.com/106294967/183713283-46f3ce59-a2a1-4527-8c13-854a58c12fb5.svg)

stakeworld commented:
Hi @rossbulat, do you know when my info will get live on the dashboard?

rossbulat commented:
It's already in the codebase and on the GitHub deployment. We do pushes to staking.Polkadot.network on a monthly basis so it'll be there soon.
